content/plugin/mux/index.mdx

```diff
@@ -0,0 +1,34 @@
+---
+page_title: 'Plugin Development - Combining and Translating Providers: Overview'
+description: >-
+  Use terraform-plugin-mux to combine and translate providers. It minimizes code changes by wrapping existing provider servers.
+---
+
+# Combining and Translating Providers
+
+The [terraform-plugin-mux](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-mux) Go module is a collection of Go packages for combining (multiplexing) and translating provider servers. It helps minimize provider code changes by wrapping existing provider servers. This functionality is based on the [Terraform Plugin Protocol](../how-terraform-works#terraform-plugin-protocol) and [terraform-plugin-go](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-go) provider servers.
+
+## Use Cases
+
+The `terraform-plugin-mux` Go module enables flexibility when you develop and maintain Terraform providers. It is especially useful when you need to upgrade an existing provider to use the plugin framework SDK or the latest version of the plugin protocol. We recommend using `terraform-plugin-mux` for the following use cases:
+
+* Combine providers to reduce maintenance burden while still supporting varying Terraform requirements or multiple provider SDK implementations. 
+* Migrate resources and data sources from [terraform-plugin-sdk/v2](../sdkv2/sdk-intro) to [terraform-plugin-framework](../framework) over time.
+* Develop with [terraform-plugin-sdk/v2](../sdkv2/sdk-intro), but require Terraform CLI 1.0 or later.
+* Develop with [terraform-plugin-framework](../framework), but support Terraform CLI 0.12 or later.
+
+## tf6muxserver
+
+Use the [tf6muxserver](./combining-protocol-version-6-providers) package to combine any number of [protocol version 6 provider servers](../how-terraform-works#protocol-version-6) into a single server.
+
+## tf5muxserver
+
+Use the [tf5muxserver](./combining-protocol-version-5-providers) package to combine any number of [protocol version 5 provider servers](../how-terraform-works#protocol-version-5) into a single server.
+
+## tf5to6server
```

Comment:
```suggestion
## Upgrade Protocol 5 Providers to Protocol 6
```

Reply:
We've referred to these pages as "Translating" in the menu bar, URLs, and generally in other places, should we flip all those as well?

Reply:
No you're right - we should keep "Translate" - Good call Brian :-)